### ISSUE TITLE
Feature request: pull path back along outer turning line

### ISSUE THREAD

**Issue by dlay97** [pyneb/pyneb#25]
Paths sometimes travel along OTL, leading to mischaracterization of exit point (see attached plot of 248-Cm). Want to cut off path at first point along OTL.
[pes.pdf](https://github.com/pyneb/pyneb/files/9123030/pes.pdf)

**Comment by kylegodbey** [comment]
Good idea, I toyed with this a while ago and the simplest thing I came to was to assign a tolerance that signaled to the action calculator to halt the integral along the path. I guess we can have a variable in the LAP object called exit point that is calculated in that way.

**Comment by dlay97** [comment]
So it seems likely that I just plotted the wrong paths there (I selected the NEB iteration with the lowest action, rather than the last iteration). The endpoints do get dragged to the correct point on the outer turning line, as shown below (this shows one path for increasing iterations). So, this is actually not an urgent issue, although perhaps we should keep an eye out for this.

[248-Cm_vs_iteration.pdf](https://github.com/pyneb/pyneb/files/9123168/248-Cm_vs_iteration.pdf)
